G2Search.py

```diff
@@ -236,12 +244,27 @@ def process_search(rowData, mappingDoc, g2Engine, searchFlags):
 
     else: # --already json or already mapped csv header
         searchStr = json.dumps(rowData)
-
     rowData['SEARCH_STRING'] = searchStr
+
+    #--get target record from truthset
+    if truthFileName:
+        try:
+            cluster_id = truthsetRecords[rowData['DATA_SOURCE']][rowData['RECORD_ID']]
+            targetRecord = truthsetTargets[cluster_id]
+            rowData['TARGET_DATA_SOURCE'] = targetRecord['DATA_SOURCE']
+            rowData['TARGET_RECORD_ID'] = targetRecord['RECORD_ID']
+        except:
+            pass
```

Comment:
This is heavily frowned upon, use instead: 

with suppress(Exception):
            cluster_id = truthsetRecords[rowData['DATA_SOURCE']][rowData['RECORD_ID']]
            targetRecord = truthsetTargets[cluster_id]
            rowData['TARGET_DATA_SOURCE'] = targetRecord['DATA_SOURCE']
            rowData['TARGET_RECORD_ID'] = targetRecord['RECORD_ID']

Reply:
ok, commented that code out.